src/dto/emissions.dto.ts

```diff
@@ -0,0 +1,34 @@
+
+import { ListDTO } from './list.dto';
+
+export class EmissionsDTO {
+  state: string;
+  facilityName: string;
+  orisCode: number;
+  unitId: string;
+  opDate: Date;
+  opHour: number;
+  opTime: number;
+  gLoad: number;
+  sLoad: number;
+  so2Mass: number;
+  so2MassMeasureFlg: string;
+  so2Rate: number;
+  sorRateMeasureFlg: string;
+  noxMass: number;
+  noxMassMeasureFlg: string;
+  noxRate: number;
+  noxRateMeasureFlg: string;
+  co2Mass: number;
+  co2MassMeasureFlg: string;
+  co2Rate: number;
+  corRateMeasureFlg: string;
+  heatInput: number;
+  primaryFuelInfo: string;
+  secondaryFuelInfo: ListDTO;

```

Comment:
Let's change the name of ListDTO to DelimitedListDTO and then create concrete types that extend DelimitedListDTO for the properties secondaryFuelInfo, so2ControlInfo, partControlInfo, noxControlInfo, hgControlInfo

Example:
export class SecondaryFuelInfoDTO extends DelimitedListDTO {
}

secondaryFuelInfo: SecondaryFuelInfoDTO

this gives us the ability to add to these types later if needed and not have to change the datatype